### PR TITLE
users: Move users/local out of the shell package

### PR DESCRIPTION
**Pull request by mvollmer**
Depends on #2577 just to avoid merge conflicts.

### CODE REVIEW

**Comment by mvollmer** [comment]
Fixed some bugs.

**Comment by mvollmer** [comment]
@dperpeet, what is the "needswork" about?

**Comment by mvollmer** [comment]
> @dperpeet, what is the "needswork" about?

Alright, found the emails. :-)

**Comment by stefwalter** [comment]
There's a 'make distcheck' error in Travis:

```
/usr/bin/xgettext: error while opening "../pkg/users/local.html.h" for reading: No such file or directory

ERROR: xgettext failed to generate PO template file. Please consult

       error message above if there is any.

make[3]: *** [cockpit.pot] Error 1
```

File: tools/cockpit.spec [hunk 1]
```diff
@@ -94,9 +94,11 @@ BuildRequires: sed
 BuildRequires: xmlto
 
 Requires: %{name}-bridge = %{version}-%{release}
+Requires: %{name}-networkmanager = %{version}-%{release}
 Requires: %{name}-ws = %{version}-%{release}
 Requires: %{name}-shell = %{version}-%{release}
 Requires: %{name}-storaged = %{version}-%{release}
+Requires: %{name}-users = %{version}-%{release}
```
Comment (stefwalter) [inline tools/cockpit.spec]: I would suggest keeping the user stuff in the shell RPM. It doesn't have any big dependencies.

Comment (mvollmer) [inline tools/cockpit.spec]: Done.

**Pull request closed without merging**